src/double_pendulum_as_single_pendulum.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from math import sin
+
+from pydrake.all import *
+from planar_rigid_body_visualizer import PlanarRigidBodyVisualizer
+from manipulator_dynamics import *
+
+class Controller(VectorSystem):
+    def __init__(self, rigid_body_tree, gravity):
+        # 4 inputs (double pend state), 2 torque outputs.
+        VectorSystem.__init__(self, 4, 2)
+        self.tree = rigid_body_tree
+        self.g = gravity
+
+    def _DoCalcVectorOutput(self, context, double_pend_state, unused, torque):
+        # Extract manipulator dynamics.
+        q = double_pend_state[:2]
+        v = double_pend_state[-2:]
+        (M,Cv,tauG,B) = manipulator_dynamics(self.tree,q,v)
+
+        # Desired pendulum parameters.
+        l = 2.; b = .1;
+
+        # Stabilize second joint.
```

Comment:
"Control gains for stabilizing second joint"? It's infer-able from the variable names but I don't know how much you want to assume of your readers